copyandpush/copyandpush.test.js

```diff
@@ -9,3 +9,13 @@ describe('copyAndPush', () => {
     expect(actual).toEqual(expected);
   });
 });
+
+describe('unchanged array', () => {
```

Comment:
This test isn't actually testing to see if the original array is unchanged. If you break it down, you're comparing `actual`, which is going to be `[1, 2, 3, 4]` here, to `expected`, which is the same, but there is no function call in this test and no original base array is being checked. See comment above for where the test should go.